packages/mobile/src/screens/sign-on-screen/components/layout.tsx

```diff
@@ -2,6 +2,7 @@ import type { ReactNode } from 'react'
 
 import { css } from '@emotion/native'
 import { Dimensions } from 'react-native'
+import { useSafeAreaInsets } from 'react-native-safe-area-context'
```

Comment:
what does this do? I'm not super familiar with it

Reply:
it basically makes sure for each device the element is in a "safe" viewable area, so for ios there is that area where the swipe bar is, it makes sure to start content above that. what this did was make sure the footer was properly spaced above the bottom ios/android elements